### PR TITLE
Add more info on missing alerts

### PR DESCRIPTION
**Pull request by joshp123**

### CODE REVIEW

**Comment by joshp123** [comment]
FYI @udh-mendix

**Comment by joshp123** [comment]
@MarkvanMents - would you mind taking a quick look, please? Thanks!

File: content/developerportal/operate/monitoring-application-health.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Any status that could cause your app to stop functioning will result in an alert
 
 There are a number of categories on the status overview that are related to various technical parts of your application environment. Any category that does not display as *OK* needs to be investigated. Each category indicates a different part of the app infrastructure that is monitored.
 
+Some alerts listed in the table below only appear when the alert is triggered. Therefore, an alert not being present in the overview is the same as the alert having an *OK* status.
+
 ### 3.1 Application Status
```
Comment (MarkvanMents) [inline content/developerportal/operate/monitoring-application-health.md:49]: @joshp123 This looks a bit weird, it looks like there should be a `### 3.n` for each of the tables, not just Application Status. Do you agree?

File: content/developerportal/operate/monitoring-application-health.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ Any status that could cause your app to stop functioning will result in an alert
 
 There are a number of categories on the status overview that are related to various technical parts of your application environment. Any category that does not display as *OK* needs to be investigated. Each category indicates a different part of the app infrastructure that is monitored.
 
+Some alerts listed in the table below only appear when the alert is triggered. Therefore, an alert not being present in the overview is the same as the alert having an *OK* status.
```
Comment (MarkvanMents) [inline content/developerportal/operate/monitoring-application-health.md:47]: @joshp123 - A presume you mean all the tables below, not just the first (Application Status) one?
```suggestion
Some alerts listed in the tables below only appear when the alert is triggered. Therefore, an alert not being present in the overview is the same as the alert having an *OK* status.
```

**Review by MarkvanMents — COMMENTED** [review]
@joshp123 
A couple of minor questions, but otherwise fine.

File: content/developerportal/operate/monitoring-application-health.md (hunk 1 above)
Comment (joshp123) [reply content/developerportal/operate/monitoring-application-health.md:49]: Hmm, it is indeed a bit weird. I think the idea is to reflect the actual alerts page, which has no subsections itself, and looks like this:



So, I would suggest leaving this as is for now.

**Comment by joshp123** [comment]
Thanks for the review, @MarkvanMents ! I have replied to your comments.

File: content/developerportal/operate/monitoring-application-health.md (hunk 1 above)
Comment (MarkvanMents) [reply content/developerportal/operate/monitoring-application-health.md:49]: OK - thanks for the explanation.

**Review by MarkvanMents — APPROVED** [review]
Shall I publish this immediately?

**Comment by joshp123** [comment]
Yes please @MarkvanMents , and thank you!

**Pull request merged**